src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/ErrorHandlingTransmissionPolicy.cs

```diff
@@ -33,7 +34,7 @@ private void HandleTransmissionSentEvent(object sender, TransmissionProcessedEve
             var webException = e.Exception as WebException;
             if (webException != null)
             {
-                HttpWebResponse httpWebResponse = webException.Response as HttpWebResponse;
+                HttpWebResponseWrapper httpWebResponse = e.Response;
```

Comment:
It always was. The change is that it will now be available on all exceptions. This allows us to use e.Response with confidence rather than webException.Response (which is not the wrapper)